WordPressKit/MediaServiceRemoteXMLRPC.m

```diff
@@ -273,12 +273,12 @@ - (RemoteMedia *)remoteMediaFromXMLRPCDictionary:(NSDictionary*)xmlRPC
     }
     remoteMedia.file = [link lastPathComponent] ?: [[xmlRPC objectForKeyPath:@"file"] lastPathComponent];
     
-    if ([xmlRPC valueForKeyPath:@"metadata.sizes.large.file"] != nil) {
-        remoteMedia.largeURL = [NSURL URLWithString: [NSString stringWithFormat:@"%@%@", remoteMedia.url.URLByDeletingLastPathComponent, [xmlRPC valueForKeyPath:@"metadata.sizes.large.file"]]];
+    if ([xmlRPC stringForKeyPath:@"metadata.sizes.large.file"] != nil) {
```

Comment:
I am not familiar with the `stringForKeyPath` method, but my understanding is that it is provided by [NSObject-SafeExpectations](https://github.com/wordpress-mobile/NSObject-SafeExpectations) and helps avoid errors while accessing attributes. It makes sense that switching to this resolves the current crash from accessing an undefined attribute. 👍🏻